tests/domain/admin_emails_test.py

```diff
@@ -10,9 +10,15 @@
 from utils.mailing import MailServiceException
 
 
+@patch('connectors.api_entreprises.requests.get')
 def test_maybe_send_offerer_validation_email_sends_email_to_pass_culture_when_objects_to_validate_and_send_email_enabled(
+        mock_api_entreprise,
         app):
     # Given
+    response_return_value = MagicMock(status_code=200, text='')
+    response_return_value.json = MagicMock(return_value={})
+    mock_api_entreprise.return_value = response_return_value
```

Comment:
```suggestion
    mock_api_entreprise.return_value = MagicMock(status_code=200, text='', json={})
```
On peut pas écrire ça en one shot ?